### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by dobooth**
- #6372
- #6374
- #6356
- #6344
- #6361
- #6342
- #6339
- #6362
- #6297
- #6326
- #6357
- #6335
- #6324
- #6334
- #6322

### CODE REVIEW

File: src/guides/v2.3/comp-mgr/marketplace/marketplace.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ functional_areas:
   - Upgrade
 ---
 
-This topic provides a simple example of getting free extensions for {{site.data.var.ce}} . For more detail, see the [Marketplace quick tour](http://docs.magento.com/marketplace/user_guide/quick-tour/install-extension.html){:target="_blank"}.
+This topic provides a simple example of getting free extensions for {{site.data.var.ce}} . For more detail, see the [Marketplace quick tour](https://docs.magento.com/marketplace/user_guide/buyers/install-extension.html){:target="_blank"}.
```
Comment (jeff-matthews) [inline src/guides/v2.3/comp-mgr/marketplace/marketplace.md]:
```suggestion
This topic provides a simple example of getting free extensions for {{site.data.var.ce}}. See the [Marketplace quick tour](https://docs.magento.com/marketplace/user_guide/buyers/install-extension.html).
```

File: src/guides/v2.3/extension-dev-guide/cli-cmds/cli-howto.md [hunk 2]
```diff
@@ -114,10 +114,16 @@ Following is a summary of the process:
    </config>
    ```
 
-1. Clean the [cache](https://glossary.magento.com/cache) and compiled code directories:
+1. Clean the [cache](https://glossary.magento.com/cache) using the command:
```
Comment (jeff-matthews) [inline src/guides/v2.3/extension-dev-guide/cli-cmds/cli-howto.md]:
```suggestion
1. Clean the [cache](https://glossary.magento.com/cache):
```

File: src/guides/v2.3/extension-dev-guide/cli-cmds/cli-howto.md [hunk 3]
```diff
@@ -114,10 +114,16 @@ Following is a summary of the process:
...
 
    ```bash
-   rm -rf var/cache/* var/page_cache/* generated/*
+   bin/magento cache:clean
+   ```
+
+1. After clean the cache, Regenerate the code using the command:
```
Comment (jeff-matthews) [inline src/guides/v2.3/extension-dev-guide/cli-cmds/cli-howto.md]:
```suggestion
1. Regenerate the code:
```

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 4]
```diff
@@ -200,6 +200,12 @@ To add new classes to the container:
 {:.bs-callout-warning}
 This method overrides existing classes.
 
+To add a new id to the container:
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md]:
```suggestion
To add a new ID to the container:
```

File: src/guides/v2.3/frontend-dev-guide/themes/theme-install.md [hunk 5]
```diff
@@ -43,7 +43,7 @@ Composer-based themes are loaded from external sources and cannot be modified di
 
 ## Install a theme as an extension
 
-If a theme is distributed on [Magento Marketplace](https://marketplace.magento.com/), install it with **Web Setup Wizard**. For details, see [Install the Extension](http://docs.magento.com/marketplace/user_guide/quick-tour/install-extension.html).
+If a theme is distributed on [Magento Marketplace](https://marketplace.magento.com/), install it with **Web Setup Wizard**. For details, see [Install the Extension](https://docs.magento.com/marketplace/user_guide/buyers/install-extension.html).
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/themes/theme-install.md]:
```suggestion
If a theme is distributed on [Magento Marketplace](https://marketplace.magento.com/), install it with the **Web Setup Wizard**. See [Install the Extension](https://docs.magento.com/marketplace/user_guide/buyers/install-extension.html).
```

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**